### PR TITLE
Add support for ~ home folder expansion

### PR DESCRIPTION
**Pull request by maxpoulin64**

### CODE REVIEW

File: src/helper.js [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+
+	if (os.homedir) {
+		home = os.homedir();
+	}
+
+	if (!home) {
+		home = process.env.HOME || "";
+	}
+
+	home = home.replace("$", "$$$$");
```
Comment (astorije) [inline src/helper.js]: Why is this necessary? Escaping? Why 4 of them lol? Would have `\$` work too (fairly common when escaping stuff)?

Comment (xPaw) [inline src/helper.js]: Because `$$$$` escapes into `$$`, which is required for the next replace to keep the final `$`.

See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter

Comment (astorije) [inline src/helper.js]: Right, I missed that part, thanks a lot for the details @xPaw :-)
